### PR TITLE
Change about , post , mission to Tinymce fields

### PR DESCRIPTION
**Pull request by Prernaswarna**

### CODE REVIEW

**Review by algomaster99 — APPROVED** [review]

File: models/group.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 from formula_one.validators.aspect_ratio import AspectRatioValidator
 from formula_one.validators.year_relation import YearRelationValidator
 
+from tinymce.models import HTMLField
```
Comment (algomaster99) [inline models/group.py]: `HTMLField` being a third-party package needs to be grouped with 
```python
import swapper
from django.contrib.contenttypes import fields as contenttypes_fields
from django.db import models
```

Remember to leave a line after the group ends.

**Review by algomaster99 — CHANGES_REQUESTED** [review]
Just one small change

File: models/group.py (hunk 1 above)
Comment (Prernaswarna) [reply models/group.py]: Hey I made the change. Is it fine ?

**Comment by algomaster99** [comment]
I am sorry I forgot to point it out before. The string representation of post model doesn't look very appealing because of the HTML tags. Maybe we should consider extracting all the raw text from there and then displaying the text.
Screenshot from 2020-09-23 17-07-50
Please refer to [this blog](https://medium.com/@jorlugaqui/how-to-strip-html-tags-from-a-string-in-python-7cb81a2bbf44) in order to do so.

File: models/post.py [hunk 2]
```diff
@@ -30,5 +33,8 @@ def __str__(self):
         """
 
         group = self.group
-        text = self.text[:80]
-        return f'{group}: {text}'
+        text = self.text[:80]        
+        clean = re.compile('<.*?>')
```
Comment (algomaster99) [inline models/post.py]: clean **before** you have truncated the text

File: models/post.py [hunk 3]
```diff
@@ -30,5 +33,8 @@ def __str__(self):
...
+        txt= re.sub(clean, '', text)
```
Comment (algomaster99) [inline models/post.py]: `text = re.sub(clean, '', text)`
Notice the space after `text`

File: models/post.py (hunk 2 above)
Comment (algomaster99) [inline models/post.py]: Why do you need to create a RegEx object? [`re.sub`](https://docs.python.org/3/library/re.html#re.sub) accepts a RegEx as a string too. Directly use `re.sub(r'<.*?>', '', text)`

**Review by algomaster99 — CHANGES_REQUESTED** [review]

**Review by algomaster99 — APPROVED** [review]

**Pull request merged**